core/templates/dev/head/components/ratingsSpec.js

```diff
@@ -116,14 +116,6 @@ describe('Ratings from frequencies directive', function() {
       '4': 0,
       '5': 0
     })).toBe(undefined);
-
-    expect(ctrlScope.computeAverageRating({
```

Comment:
You can write it to check that after one rating the stars are shown, which will fail if the acceptable number is raised.